backend/src/main/java/com/handong/rebon/review/domain/Review.java

```diff
@@ -34,4 +41,31 @@
     @OneToMany(mappedBy = "review")
     private List<Empathy> empathies;
```

Comment:
```suggestion
    @OneToMany(mappedBy = "review")
    private List<Empathy> empathies = new ArrayList<>();
```

컬렉션의 경우 null문제가 일어날 수 있어서 필드에서 바로 초기화해주는 것이 좋다고 합니다.
https://velog.io/@yebali/Spring-JPA-Entity%EC%84%A4%EA%B3%84-%EC%8B%9C-%EC%A3%BC%EC%9D%98%EC%A0%90

Reply:
감사합니다!